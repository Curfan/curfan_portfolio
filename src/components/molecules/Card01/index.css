.card01 {
	& > img {
		width: 340px;
	}

	& > .title {
		font-size: 18px;
		line-height: 21px;
		letter-spacing: 0.1em;
		margin-top: 12px;
	}

	& > .description {
		font-size: 14px;
		line-height: 17px;
		letter-spacing: 0.1em;
		margin-top: 8px;
	}
}
