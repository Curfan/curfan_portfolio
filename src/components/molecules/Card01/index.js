import React from 'react';
import classnames from 'classnames';

import styles from './index.css';

const Card01 = ({ className, photo, title, description }) => (
	<div className={classnames(styles.card01, className)}>
		<img src={photo} alt="work" />
		<div className={styles.title}>{title}</div>
		<div className={styles.description}>{description}</div>
	</div>
);

export default Card01;
