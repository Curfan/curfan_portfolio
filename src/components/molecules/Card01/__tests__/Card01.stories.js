import React from 'react';
import { storiesOf } from '@storybook/react';
import { text } from '@storybook/addon-knobs/react';

import Card01 from 'components/molecules/Card01';

const stories = storiesOf('molecules/Card01', module);

stories.add('__interactive', () => (
	<div style={{ maxWidth: '240px' }}>
		<Card01
			photo={text('photo', 'https://fakeimg.pl/300/')}
			title={text('title', 'Product Name')}
			description={text('description', 'Task')}
		/>
	</div>
));
