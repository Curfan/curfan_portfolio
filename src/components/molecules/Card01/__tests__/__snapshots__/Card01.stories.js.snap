// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__interactive 1`] = `
<DocumentFragment>
  <div
    class="card01"
  >
    <img
      alt="work"
      src="https://fakeimg.pl/300/"
    />
    <div
      class="title"
    >
      Product Name
    </div>
    <div
      class="description"
    >
      Task
    </div>
  </div>
</DocumentFragment>
`;
